src/Bridge/Symfony/Resources/config/form.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <services>
+        <service id="greg0ire_enum.form.type.enum" class="Greg0ire\Enum\Bridge\Symfony\Form\Type\EnumType">
```

Comment:
I think the FQCN should be used as service id, shouldn't it?

Reply:
I think so. Fixed.